src/test/java/homework14/homework/com/DemoQaElementsTest.java

```diff
@@ -0,0 +1,163 @@
+import org.openqa.selenium.interactions.Actions;
+import java.time.Duration;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DemoQaElementsTest {
+
+    private static final By ELEMENTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Elements']");
+    private static final By CHECK_BOX_MENU_ITEM = By.id("item-1");
+    private static final By CHECK_BOX_TITLE = By.className("text-center");
+    private static final By HOME_CHECK_BOX = By.className("rct-checkbox");
+    private static final By HOME_CHECK_BOX_RESULT = By.id("result");
+    private static final String EXPECTED_MESSAGE_HOME_CHECK_BOX = "You have selected :\n" + "home\n" + "desktop\n" +
+            "notes\n" + "commands\n" + "documents\n" + "workspace\n" + "react\n" + "angular\n" + "veu\n" + "office\n" +
+            "public\n" + "private\n" + "classified\n" + "general\n" + "downloads\n" + "wordFile\n" + "excelFile";
+    private static final By BUTTONS_MENU_ITEM = By.id("item-4");
+    private static final By DOUBLE_CLICK_BUTTON = By.id("doubleClickBtn");
+    private static final By MESSAGE_DOUBLE_CLICK_BUTTON = By.id("doubleClickMessage");
+    private static final String EXPECTED_MESSAGE_DOUBLE_CLICK_BUTTON = "You have done a double click";
+    private static final By RIGHT_CLICK_BUTTON = By.id("rightClickBtn");
+    private static final By MESSAGE_RIGHT_CLICK_BUTTON = By.id("rightClickMessage");
+    private static final String EXPECTED_MESSAGE_RIGHT_CLICK_BUTTON = "You have done a right click";
+    private static final By LINKS_MENU_ITEM = By.id("item-5");
+    private static final By NOT_FOUND_LINK = By.id("invalid-url");
+    private static final By MESSAGE_NOT_FOUND_LINK = By.id("linkResponse");
+    private static final String EXPECTED_MESSAGE_NOT_FOUND_LINK = "Link has responded with staus 404 and status text Not Found";
+    private static final By WEB_TABLES_MENU_ITEM = By.id("item-3");
+    private static final By ADD_NEW_RECORD_BUTTON = By.id("addNewRecordButton");
+    private static final By PERMANENT_FIRST_NAME = By.id("firstName");
+    private static final String FIRST_NAME = "Thomas";
+    private static final By PERMANENT_LAST_NAME = By.id("lastName");
+    private static final String LAST_NAME = "Andersen";
+    private static final By PERMANENT_USER_EMAIL = By.id("userEmail");
+    private static final String USER_EMAIL = "andersen@gmail.com";
+    private static final By PERMANENT_USER_AGE = By.id("age");
+    private static final String USER_AGE = "34";
+    private static final By PERMANENT_SALARY = By.id("salary");
+    private static final String SALARY = "55000";
+    private static final By PERMANENT_DEPARTMENT = By.id("department");
+    private static final String DEPARTMENT = "qa automation";
+    private static final By SUBMIT_NEW_RECORD_TABLE_BUTTON = By.id("submit");
+    private static final By ADDED_RECORD = By.xpath("//*[@id=\"app\"]/div/div/div/div[2]/div[2]/div[3]/div[1]/div[2]/div[4]/div");
```

Comment:
looks like this is not the better locator. please provide the page where it is located